lean4/src/putnam_1989_a6.lean

```diff
@@ -17,6 +17,8 @@ theorem putnam_1989_a6
 (F : Type*) [Field F] [Fintype F]
 (hF : Fintype.card F = 2)
 (α : PowerSeries F)
-(hα : ∀ n : ℕ, let bin := [1] ++ (digits 2 n) ++ [1]; PowerSeries.coeff F n α = ite (∀ i j : Fin bin.length, i < j → bin.get i = 1 → bin.get j = 1 → (∀ k, i < k → k < j → bin.get k = 0) → Even ((j : ℕ) - (i : ℕ) - 1)) 1 0)
```

Comment:
This can be fixed with `set_option synthInstance.maxSize $random_number`, which I am reporting now...

Reply:
https://leanprover.zulipchat.com/#narrow/channel/270676-lean4/topic/Success.20is.20not.20monotone.20in.20.60synthInstance.2EmaxSize.60/near/491823649